sw/device/silicon_creator/mask_rom/mask_rom.ld

```diff
@@ -105,13 +105,28 @@ SECTIONS {
   } > rom
 
   /**
-   * Mutable data section, at the bottom of ram_main. This will be initialized
-   * from rom at runtime by the CRT.
+   * Critical static data that is accessible by both the mask ROM and the ROM
+   * extension.
    *
-   * Load this by copying the bytes from [_data_init_start, _data_init_end] into
-   * the range [_data_start, _data_end].
+   * Each variable added to .critical_static must be in its own subsection
+   * named after the variable.
+   *
+   * This data is not initialized during CRT.
```

Comment:
Nit: I think it's worth explaining why, and that this is why the section is of `NOLOAD` type.

Reply:
Done.